tests/integration/test_tls.py

```diff
@@ -106,6 +110,10 @@ async def test_tls_enabled(ops_test: OpsTest) -> None:
         )
         change_primary_start_timeout(ops_test, primary, 0)
 
+        # Pause Patroni so it doesn't wipe the custom changes
+        await change_patroni_setting(ops_test, "pause", True, use_random_unit=True, tls=True)
```

Comment:
Pause and upause Patroni, instead of trying to tweak the `loop_wait`

Reply:
Great solution!